wavelink/ext/spotify/__init__.py

```diff
@@ -445,31 +445,16 @@ async def _search(self,
         if data['type'] == 'track':
             return SpotifyTrack(data)
 
+
         elif data['type'] == 'album':
-            album_data: dict[str, Any]= {
-                                        'album_type': data['album_type'],
-                                        'artists': data['artists'],
-                                        'available_markets': data['available_markets'],
-                                        'external_urls': data['external_urls'],
-                                        'href': data['href'],
-                                        'id': data['id'],
-                                        'images': data['images'],
-                                        'name': data['name'],
-                                        'release_date': data['release_date'],
-                                        'release_date_precision': data['release_date_precision'],
-                                        'total_tracks': data['total_tracks'],
-                                        'type': data['type'],
-                                        'uri': data['uri'],
-                                        }
             tracks = []
```

Comment:
Did you mean to delete the album data? If so, then why have you chosen to not have this information?

Reply:
I did. While this data was not contained in SimplifiedTrackObjects and had to be added, it is included in full TrackObjects.

Reply:
I believe that not all of these keys here are available on the SpotifyTrack objects.
e.g. album_type, release_date, release_date_precision, total_tracks and href for the album.

Reply:
They are all listed as required keys under a SpotifyTrack's album key, so no data should be lost. However, I agree that this is over correcting for the issue of searching for "None" when ISRC is None.